aerleon/lib/junipersrx.py

```diff
@@ -675,7 +677,7 @@ def Chunks(addresses):
             del terms[:]
             terms.extend(expanded_terms)
 
-    def _BuildPort(self, ports):
+    def _BuildPort(self, ports: List[Union[Any, Tuple[int, int]]]) -> List[Union[Any, str]]:
```

Comment:
Both "Any" hints can be removed. The type for "ports" is `List[Tuple[int, int]]` and the return type is `List[str]`

Reply:
done